ocs_ci/deployment/ibm.py

```diff
@@ -0,0 +1,229 @@
+"""
+This module implements the OCS deployment for IBM Power platform
+Base code in deployment.py contains the required changes to keep
+code duplication to minimum. Only destroy_ocs is retained here.
+"""
+import logging
+
+from ocs_ci.deployment.deployment import Deployment
+
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs.exceptions import (
+    CommandFailed
+)
+# from ocs_ci.ocs.resources.storage_cluster import change_noobaa_endpoints_count
+from ocs_ci.utility.utils import (
+    run_cmd
+)
+
+logger = logging.getLogger(__name__)
+
+
+class IBMDeployment(Deployment):
+    """
+    Implementation of Deploy for IBM Power architecture
+    """
+
+    def destroy_ocs(self):
```

Comment:
I think this is quite the same as https://github.com/red-hat-storage/ocs-ci/blob/2c356000575f4127782ee383d2dfc0593f04500a/ocs_ci/ocs/uninstall.py#L145

Reply:
When I try --teardown option while running run-ci, I do not see uninstall_ocs or ocs-ci/ocs_ci/ocs/uninstall is involved. I will look at this in future. We may not use ocs teardown much, may remove OCP itself.